src/views/NoteView.vue

```diff
@@ -1,5 +1,165 @@
 <template>
-  <h1>Note View</h1>
+  <div class="note-view">
+    <div class="header-controls">
+      <div class="note-info">
+        <div class="note-channel">
+          <span>channel: </span>
+          <span class="note-channel">{{ note.channel }}</span>
+        </div>
+        <div class="note-tag">
+          <span>tag: </span>
+          <span class="note-tag">{{ note.tag }}</span>
+        </div>
+      </div>
+      <div class="note-actions">
+        <button class="note-action-button">ホームへ戻る</button>
+        <button class="note-action-button">編集</button>
+      </div>
```

Comment:
router にリンクを入れる関数を `@click="function()"` に入れて、クリックで発火するようにしましょう
変数名はなんかいい感じので

```suggestion
      <div class="note-actions">
        <button class="note-action-button" @click="goHome">ホームへ戻る</button>
        <button class="note-action-button" @click="goEdit">編集</button>
      </div>
```